src/integrationTest/groovy/wooga/gradle/IntegrationSpec.groovy

```diff
@@ -23,9 +23,10 @@ import org.junit.Rule
 import org.junit.contrib.java.lang.system.EnvironmentVariables
 import org.junit.contrib.java.lang.system.ProvideSystemProperty
 import wooga.gradle.unity.utils.PlatformUtilsImpl
+import wooga.gradle.utils.PropertyUtilsImpl
 
 class IntegrationSpec extends nebula.test.IntegrationSpec
-        implements PropertyUtils, PlatformUtilsImpl {
+        implements PropertyUtilsImpl, PlatformUtilsImpl {
```

Comment:
This `PropertyUtilsImpl` is a trait I hope? Because I get slight nervious ticks when seeing an Interface called Impl :)

Reply:
Yes, it's a trait.